src/emulator/storage/rules/runtime.ts

```diff
@@ -250,7 +265,7 @@ export class StorageRulesRuntime {
     };
 
     const response = (await this._sendRequest(
-      runtimeActionRequest
+      runtimeActionRequest, this._childprocess as ChildProcess
```

Comment:
if you want to unwrap nullable types, you can use the `!` operated, e.g. `this._childProcess!` but this will throw an unhandled `Error` if `this._childProcess === null | undefined`

Reply:
Thanks for the extra info turns out I can just use `this` actually so that simplifies this